src/pop_protocol.erl

```diff
@@ -289,17 +284,24 @@ resolve_fork_select_branch(B1, B2, TreeData) ->
     end.
 
 resolve_fork_same_parent(B1, B2) ->
-    CD1 = maps:get(consensus_data, B1),
-    CD2 = maps:get(consensus_data, B2),
-
-    T1 = maps:get(timestamp, CD1),
-    T2 = maps:get(timestamp, CD2),
+    T1 = get_block_cd(timestamp, B1),
+    T2 = get_block_cd(timestamp, B2),
 
     if 
 	T1 < T2 ->
 	    first;
 	T2 < T1 ->
-	    second
```

Comment:
what happened before when T1 == T2?

Reply:
(before, you would never encounter t1 = t2)